src/OpenLoco/src/Windows/ObjectSelectionWindow.cpp

```diff
@@ -99,21 +159,112 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
 
     static WindowEventList _events;
 
-    // 0x00473154
-    static void sub_473154(Window* self)
+    // 0x0047322A
+    static void rotateTabs(uint8_t newStartPosition)
     {
-        registers regs;
-        regs.esi = X86Pointer(self);
-        call(0x00473154, regs);
+        TabPosition newPosInfo[36] = {};
```

Comment:
Is there any reason we can't just std::rotate this? would obviously need to fix the row after the rotate but it would be simpler.

Reply:
I looked at that, but because of the `0xFF` sentinel I don't think we can use `std::rotate`...